library/core/src/fmt/rt.rs

```diff
@@ -135,11 +166,6 @@ impl Argument<'_> {
     /// # Safety
     ///
     /// This argument must actually be a placeholder argument.
-    ///
-    // FIXME: Transmuting formatter in new and indirectly branching to/calling
-    // it here is an explicit CFI violation.
-    #[allow(inline_no_sanitize)]
-    #[no_sanitize(cfi, kcfi)]
```

Comment:
`#![feature(no_sanitize)]` can now be removed from `library/core/src/lib.rs`

(sorry, I can't leave a suggestion for this)